tests/ast/data/functions.test

```diff
@@ -744,7 +744,7 @@ df72 = df.select(seq8(324))
 
```

Comment:
Overhauled a lot of the functions, their unparsed repro is now much closer to the user submitted code.